index.html

```diff
@@ -713,29 +713,29 @@ <h2>Trust Model</h2>
       <h2>Basic Concepts</h2>
 
       <p>
-This section introduces some basic concepts for the specification, in
+This section introduces some basic concepts for the specification, in 
```

Comment:
Can you set your editor to avoid end-of-line whitespace and fix this whole file?

Reply:
This is likely due to manu's update.  This probably needs a rebase on top of master.

Reply:
I think a rebase is needed?  @msporny did a commit (https://github.com/w3c/vc-data-model/commit/bdce6b93c33be383f9bfb53c20969b54f743e87f) that removed lots of whitespace, this commit is adding that back.  And at least one typo ("verififiable") is added back too.  Not sure what happened there.  Might be easier to handle and review if the relevant commits are rebased on top of latest master?